src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -155,6 +155,19 @@ public AcquireTokenForClientParameterBuilder WithFmiPath(string pathSuffix)
             return this;
         }
 
+        /// <summary>
+        /// Adds custom HTTP headers to the token request for client credential flows.
+        /// </summary>
+        /// <remarks>
+        /// Headers set here will be merged into the request-level headers; user-specified values win.
+        /// </remarks>
+        public AcquireTokenForClientParameterBuilder WithExtraHttpHeaders(
+            IDictionary<string, string> extraHttpHeaders)
+        {
+            CommonParameters.ExtraHttpHeaders = extraHttpHeaders;
```

Comment:
I don't think this is needed @gladjohn . It's already exposed on all param builders.

Reply:
<img width="1611" height="517" alt="image" src="https://github.com/user-attachments/assets/a2c81119-b33f-4055-bb3d-165601afa01f" />

Reply:
> It's in the "advanced" namespace, so maybe you want to move it (i.e. copy it) in the "extensibiltiy" namepsace

added now